desk/sur/vitals.hoon

```diff
@@ -41,4 +41,17 @@
   $:  =ship
       =result
   ==
+::
++$  ping
+  $:  version=@
+      ships=(set ship)
+      nonce=@ud
+      $=  plan
+      $~  [%nat ~]
+      $%  [%nat ~]
+          [%pub ip=(unit @t)]
+          [%off ~]
+          [%one ~]
+      ==
```

Comment:
I know `/app/ping` doesn't have them either rn, but I would like to get one-liners for each of these cases describing what they signify.